easybuild/tools/config.py

```diff
@@ -448,16 +450,6 @@ def get_log_filename(name, version, add_salt=False):
     return filepath
 
 
-def read_only_installdir():
-    """
-    Return whether installation dir should be fully read-only after installation.
-    """
```

Comment:
I could, but this is not considered to be part of the 'external' API, i.e. what easyblocks should be using. So it's OK to drop this.